### DISCUSSION TITLE
Lesson 9: register new upkeep

### DISCUSSION THREAD

**Discussion by Eshan-Sharma** [Cyfrin/foundry-full-course-cu#1724] [Q&A] [answered]
After the contract is created and verified on etherscan. What will be the contract call to be registered for new upkeep?
![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/43044334/2ded2c5a-7b70-4288-a4e1-907d2bdd55b7)

If we select ```performUpkeep```, what will be function inputs?

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/43044334/5eff5eeb-8050-4c65-93ec-d38e4dfdbd64)

**Comment by EngrPips** [comment]
Hello @Eshan-Sharma, If you are following Patrrick's course, then I am convinced he explained everything in the course.

**Reply by Eshan-Sharma** [reply]
yes I am following him but this part was not present in the course. This is from 34. Deploy the lottery on the testnet pt.2

**Reply by EngrPips** [reply, marked as answer]
I will say keep following the course because he touched on everything in the course. I have seen scenarios where people say some part was not explained, and after they keep watching for a couple of minutes, they find out every part was explained, and they need to be a little bit more patient

**Reply by Eshan-Sharma** [reply]
Thank you! It was right there and I missed it

**Reply by EngrPips** [reply]
Awesome, I am glad you finally found it. Keep crushing it.
